common/network-shuffle/src/test/java/org/apache/spark/network/shuffle/ExternalShuffleBlockResolverSuite.java

```diff
@@ -146,12 +146,25 @@ public void jsonSerializationOfExecutorRegistration() throws IOException {
 
   @Test
   public void testNormalizeAndInternPathname() {
-    assertPathsMatch("/foo", "bar", "baz", "/foo/bar/baz");
-    assertPathsMatch("//foo/", "bar/", "//baz", "/foo/bar/baz");
-    assertPathsMatch("foo", "bar", "baz///", "foo/bar/baz");
-    assertPathsMatch("/foo/", "/bar//", "/baz", "/foo/bar/baz");
-    assertPathsMatch("/", "", "", "/");
-    assertPathsMatch("/", "/", "/", "/");
+    assertPathsMatch("/foo", "bar", "baz",
+            File.separator + "foo" + File.separator + "bar" + File.separator + "baz");
+    assertPathsMatch("//foo/", "bar/", "//baz",
+            File.separator + "foo" + File.separator + "bar" + File.separator + "baz");
+    assertPathsMatch("foo", "bar", "baz///",
+            "foo" + File.separator + "bar" + File.separator + "baz");
+    assertPathsMatch("/foo/", "/bar//", "/baz",
+            File.separator + "foo" + File.separator + "bar" + File.separator + "baz");
+    assertPathsMatch("/", "", "",
+            File.separator);
+    assertPathsMatch("/", "/", "/",
+            File.separator);
+    if (ExecutorDiskUtils.isWindows()) {
+      assertPathsMatch("/foo\\/", "bar", "baz",
+              File.separator + "foo" + File.separator + "bar" + File.separator + "baz");
```

Comment:
Nit: too much indent

Reply:
reduced to 2 space.
BTW, consider add this rule to check-style? I also see some 4 space indent.

Reply:
I agree it would better to have it as rule but from the start as now there are many violations. If you would correct all of them with 1 commit than git blame would be a bit harder to use a blame would show your commit first on a such a line where we definitely would be more interested about the feature/bug which touched it before.